src/main/java/lotto/domain/Lotto.java

```diff
@@ -9,11 +9,16 @@ public class Lotto {
 
     private static final int LOTTO_NUMBER_AMOUNT = 6;
 
-    private final List<LottoNumber> numbers;
+    private final LottoNumbers numbers;
+
+    public Lotto(final LottoNumbers numbers) {
+        validateOrThrow(numbers.getLottoNumbers());
+        this.numbers = numbers;
+    }
 
     public Lotto(final List<LottoNumber> numbers) {
         validateOrThrow(numbers);
-        this.numbers = numbers;
+        this.numbers = new LottoNumbers(numbers);
```

Comment:
기존 생성자를 호출하는 방식으로 구조를 변경해보는 건 어떨까요?
```suggestion
        this(new LottoNumbers(numbers));
```

Reply:
반영하겠습니다~